### PR TITLE
Remove redundant `HTTPError` cast check in web server

### PR DESCRIPTION
**Pull request by kleimkuhler**
Signed-off-by: Kevin Leimkuhler <kleimkuhler@icloud.com>

### CODE REVIEW

File: web/srv/api_handlers.go [hunk 1]
```diff
@@ -219,7 +216,7 @@ func (h *handler) handleAPITap(w http.ResponseWriter, req *http.Request, p httpr
 	}
 
 	if messageType != websocket.TextMessage {
-		websocketError(ws, websocket.CloseUnsupportedData, errors.New("MessageType not supported"))
+		websocketError(ws, websocket.CloseUnsupportedData, errors.New("messageType not supported"))
```
Comment (kleimkuhler) [inline web/srv/api_handlers.go]: Removed the capitalization for correct style

File: web/srv/api_handlers.go [hunk 2]
```diff
@@ -243,7 +240,8 @@ func (h *handler) handleAPITap(w http.ResponseWriter, req *http.Request, p httpr
 			// socket with `ClosePolicyViolation` status code so that the error
 			// renders without the error prefix in the banner
 			if httpErr, ok := err.(protohttp.HTTPError); ok && httpErr.Code == http.StatusForbidden {
-				websocketError(ws, websocket.ClosePolicyViolation, err)
+				msg := fmt.Sprintf("Missing authorization, visit %s to remedy", tap.TapRbacURL)
+				websocketError(ws, websocket.ClosePolicyViolation, errors.New(msg))
```
Comment (siggy) [inline web/srv/api_handlers.go]:
```
err := fmt.Errorf("missing authorization, visit %s to remedy", tap.TapRbacURL)
websocketError(ws, websocket.ClosePolicyViolation, err)
```

File: web/srv/api_handlers.go [hunk 3]
```diff
@@ -21,6 +21,8 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+// Control Frame payload size can be no bigger than 125 bytes. 2 bytes are
+// reserverd for the status code when formatting the message.
```
Comment (siggy) [inline web/srv/api_handlers.go]: i think the 2 bytes are for payload size:
https://github.com/gorilla/websocket/blob/6a67f44b69004415802d2e678e483065b8fff6c8/conn.go#L1159-L1162

```
// Control Frame payload size can be no bigger than 125 bytes. 2 bytes are
// reserved for the payload size when formatting the message.
```

Comment (kleimkuhler) [reply web/srv/api_handlers.go]: @siggy `FormatCloseMessage` takes the status code we are closing the socket with and the message. It copies the close code into the first two bytes of the buffer, and then the message into the rest. That message is what Gorilla validates to be no longer than 125 bytes.

Maybe I'm missing something, but it doesn't seem like is related to the size of the payload?

File: web/srv/api_handlers.go (hunk 2 above)
Comment (kleimkuhler) [reply web/srv/api_handlers.go]: @siggy I like this change, but I think "Missing" should still be capitalized. We display this error with no prefix in the banner, so it's always going to be the first word and may look weird lowercase.

Comment (siggy) [reply web/srv/api_handlers.go]: Agree that capitalizing messages in the banner is preferable. I don't think the code at this layer should be concerned about that. Better to consistently return well-formed Go error messages, and let the presentation layer decide how to render them.

File: web/srv/api_handlers.go (hunk 3 above)
Comment (siggy) [reply web/srv/api_handlers.go]: Ah you're right, I misread `FormatCloseMessage`.

File: web/srv/api_handlers.go (hunk 2 above)
Comment (kleimkuhler) [reply web/srv/api_handlers.go]: Yep that is a good point. I'll remove the capitalization and let the open issue on how we want to present this error handle that!

**Pull request merged**